src/checkpoint1/checkpoint1.html

```diff
@@ -0,0 +1,78 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>checkpoint 1</title>
+    <link rel="stylesheet" href="./reset.css">
+    <link rel="stylesheet" href="./checkpoint1.css">
+    <script src="https://kit.fontawesome.com/ceff186d6a.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <header>
+        <i class="far fa-envelope fa-lg"></i>
+        <i class="fas fa-list fa-lg"></i>
+        <i class="far fa-bell fa-lg"></i>
+        <i class="far fa-user-circle fa-lg"></i>
+    </header>
+
+    <div class="title">HYBRID</div>
+    
+    <aside>
+        <i class="far fa-envelope fa-lg"> Dashboard</i>
+        <i class="fas fa-list fa-lg"> UI Elements</i>
+        <i class="far fa-bell fa-lg"> Charts</i>
+        <i class="far fa-user-circle fa-lg"> Tabs & Panels</i>
+        <i class="far fa-envelope fa-lg"> Tables</i>
+        <i class="fas fa-list fa-lg"> Forms</i>
+        <i class="far fa-bell fa-lg"> List</i>
+        <i class="far fa-user-circle fa-lg"> Account</i>
+    </aside>
+    
+    <main>
+        
+    </main>
+
+    <div class="dash">Dashboard  <h6>Summary of your app</h6></div>
+
+    <div class="title"></div>
+
+    <nav>
+        <h4>Home /</h4>
+        <h4>Library /</h4>
+        <h4>Data</h4>
+    </nav>
+        <div class="one">
+            <i class="far fa-eye fa-lg"></i>
+            <h3>8,457</h3>
+            <h5>Daily Visits</h5>
+        </div>
```

Comment:
so i noticed you had a margin for the h3 and i'm assuming you wanted more space between the icon and h3 and h5. margin works. but you also used flexbox. with one tweak to this code and a change in the css, i think you can achieve what you wanted. 
remember: group like items. so I'd put h3 and h5 in a div. so now there are only two elements inside your .one class.
`<div class="one">`
            `<i class="far fa-eye fa-lg"></i>`
            `<div>`
                `<h3>8,457</h3>`
                `<h5>Daily Visits</h5>`
           ` </div>`
       ` </div>`
and in the .one class, change `justify-content: space-around;` to `justify-content: space-between;`. (don't forget to add `padding: 10px;`. 
now if you want more space between, i'd add a little margin-top on the div.
_first div has padding, the div, and space-between._
<img width="329" alt="Screen Shot 2020-02-07 at 4 24 51 PM" src="https://user-images.githubusercontent.com/6327260/74070613-22d62300-49c7-11ea-8db6-ef0e81f90493.png">